<?xml version="1.0" encoding="UTF-8"?>
<ui version="5.0">
 <class>LidarViewMainWindow</class>
 <widget class="QMainWindow" name="LidarViewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>LidarView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/lvResources.qrc">
    <normaloff>:/lvResources/Icons/logo.png</normaloff>:/lvResources/Icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="pqViewManager" name="MultiViewManager" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFileBase"/>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEditBase"/>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>&amp;Sources</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>&amp;Filters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMacros">
    <property name="title">
     <string>&amp;Macros</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelpBase"/>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSources"/>
   <addaction name="menuFilters"/>
   <addaction name="menuTools"/>
   <addaction name="menuMacros"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="lqStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="colorMapEditorDock">
   <property name="windowTitle">
    <string>Color Map Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqColorMapEditor" name="colorMapEditorPanel"/>
  </widget>
  <widget class="QDockWidget" name="lidarPlayerDock">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>LiDAR Player</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="lqLivePlayerWidget" name="livePlayerPanel"/>
  </widget>
  <widget class="QDockWidget" name="pipelineBrowserDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPipelineBrowserWidget" name="pipelineBrowser"/>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="propertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="memoryInspectorDock">
   <property name="windowTitle">
    <string>Memory Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqMemoryInspectorPanel" name="memoryInspectorPanel"/>
  </widget>
  <widget class="QDockWidget" name="informationDock">
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqProxyInformationWidget" name="proxyInformationWidget"/>
  </widget>
  <widget class="QDockWidget" name="displayPropertiesDock">
   <property name="windowTitle">
    <string>Display</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="displayPropertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="viewPropertiesDock">
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="viewPropertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="outputWidgetDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Output Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqOutputWidget" name="outputWidget">
    <property name="settingsKey" stdset="0">
     <string>OutputMessages</string>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="findDataDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Find Data</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="findDataScrollArea">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="pqFindDataWidget" name="findDataWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>68</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="pythonShellDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Python Shell</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="pythonShellDummy">
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Python support not available!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqViewManager</class>
   <extends>QWidget</extends>
   <header>pqViewManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqColorMapEditor</class>
   <extends>QWidget</extends>
   <header>pqColorMapEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lqLivePlayerWidget</class>
   <extends>QWidget</extends>
   <header>lqLivePlayerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lqStatusBar</class>
   <extends>QStatusBar</extends>
   <header>lqStatusBar.h</header>
  </customwidget>
  <customwidget>
   <class>pqPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>pqPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqProxyInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqProxyInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqMemoryInspectorPanel</class>
   <extends>QWidget</extends>
   <header>pqMemoryInspectorPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqFindDataWidget</class>
   <extends>QWidget</extends>
   <header>pqFindDataWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqOutputWidget</class>
   <extends>QWidget</extends>
   <header>pqOutputWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/lvResources.qrc"/>
  <include location="../../LVCore/Qt/ApplicationComponents/lqResources.qrc"/>
 </resources>
 <connections/>
</ui>
